assets/app/view/spreadsheet.rb

```diff
@@ -96,10 +97,20 @@ def render_title
     end
 
     def render_corporations
-      @game.corporations.map { |c| render_corporation(c) }
+      current_round = [@game.turn]
+      current_round += [@game.round.round_num] if @game.round.name == 'Operating Round'
```

Comment:
current_round << @game.round.round_num if ...
current_round = current_round.join('.')

Reply:
for the join I would rather do it in the `corporation_operated?` function. Keep the compare logic in 1 place :)